_posts/en/posts/2016-01-24-clarification.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Clarifying Communications
+name: clarifying-communications
+type: post
+permalink: /en/2015/01/24/clarification
+share: true
+---
+The Bitcoin Core team would like to address some recent misunderstandings about the relationship between Bitcoin Core and public venues for information and discussion about Bitcoin itself.
+
+We encourage the Bitcoin community to civilly discuss and critique any and all changes to the system. However, we also recognize that venues have specific purposes and policies that must be respected. Our wider community can become extremely excited and heated when it comes to discussing Bitcoin. At the same time, there is universal agreement to not engage in brigading, to reject participation in denial-of-service attacks, and to assume good faith in absence of reason to believe otherwise.
```

Comment:
"universal agreement" is intended to mean that it is a widely considered bad behaviour, that it is a reasonable assumption of societal norms. Maybe it would clarify, without changing the meaning to say "it is generally considered wrong to...".